app/frontend/cypress/component/App.cy.tsx

```diff
@@ -77,7 +78,7 @@ describe('App', () => {
       cy.findByRole('button', { name: /side-by-side/i }).should('be.disabled');
     });
 
-    it('should disable arrow buttons while loading next spec', () => {
+    it.skip('should disable arrow buttons while loading next spec', () => {
```

Comment:
skipping as this functionality does not exist anymore. 
we disable arrow only when we currently on the first or the last page

Reply:
Let's delete this test then!